packages/web-sdk/src/metas/k6/meta.ts

```diff
@@ -0,0 +1,11 @@
+import { isObject } from '@grafana/faro-core';
+import type { Meta, MetaItem } from '@grafana/faro-core';
+
+export const k6Meta: MetaItem<Pick<Meta, 'k6'>> = () => {
+  const isK6Browser = isObject((window as any).k6);
```

Comment:
Does this need a condition? I.e: only add this meta if true?

Reply:
Not really because we only the meta if Faro is running in k5 browser.
It's redundant, will remove it.